### PR TITLE
Ability to set/unset async commit listener finalization via HU.

### PR DESCRIPTION
**Pull request by dsilvam**

### CODE REVIEW

File: dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java [hunk 1]
```diff
@@ -732,6 +734,14 @@ public static void setTransactionListenersStatus(TransactionListenerStatus statu
 		listenersStatus.set(status);
 	}
 
+	public static ThreadLocal<Boolean> getAsyncCommitListenersFinalization() {
+		return asyncCommitListenersFinalization;
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java]: won't be better just to return the asyncCommitListenersFinalization.get() in order to return just the boolean

**Review by jdotcms — COMMENTED** [review]
just a comment

File: dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java (hunk 1 above)
Comment (dsilvam) [reply dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java]: agree, will change in master PR

Comment (dsilvam) [reply dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java]: @jdotcms commited to PR for master

**Pull request merged**